src/bootstrap/src/core/config/config.rs

```diff
@@ -1608,6 +1607,14 @@ impl Config {
             toml.profile = Some("dist".into());
         }
 
+        for include_path in toml.include.clone().unwrap_or_default().iter().rev() {
```

Comment:
It seems like you did it in the opposite order than I suggested :laughing: So if I understand it correctly, instead of doing profile -> include with override, you went for include -> profile with ignoreduplicate. Hmm, I guess that also works, as long as you iterate in reverse order of includes.

Could you please add comments to these include loops that explain the behavior of the parent/child, left/right and include/profile precedences? It's quite difficult to understand from the code itself that `rev()`, `IgnoreDuplicate`, and the include/profile order is load-bearing.

Reply:
I think extensions should override profiles since users create them manually and user-defined settings should always take precedence over standard defaults.  

Sure, I can add a comment.

Reply:
Yeah, I also think that extensions should override profiles, I just meant "different source code order", since you change the iteration order and the merge mode, but that doesn't really matter.